cqlsh/__main__.py

```diff
@@ -0,0 +1,10 @@
+import os
+import sys
+from cqlsh.cqlsh import main as cqlsh_main
+from cqlsh.cqlsh import read_options
+
+def main():
+    cqlsh_main(*read_options(sys.argv[1:], os.environ))
```

Comment:
This always return None...

Reply:
Would you be able to suggest a PR?  That would make it easier to understand what you are suggesting.  There are non-zero exit codes it can return; cqlsh.py has eight sys.exit() calls defined in the Apache code in cqlsh.py.

```
% cqlsh -u foo -p none
Connection error: ('Unable to connect to any servers', {'127.0.0.1:9042': AuthenticationFailed('Failed to authenticate to 127.0.0.1:9042: Error from server: code=0100 [Bad credentials] message="Provided username foo and/or password are incorrect"')})
% echo $?
1

% cqlsh 
Connected to Test Cluster at 127.0.0.1:9042
[cqlsh 6.0.0 | Cassandra 4.0.5 | CQL spec 3.4.5 | Native protocol v5]
cassandra@cqlsh> exit
% echo $?
0
```